uk.org.greenend.chiark.sgtatham.puzzles.yaml

```diff
@@ -37,7 +37,9 @@ modules:
     buildsystem: cmake-ninja
     post-install:
       - install -Dm644 uk.org.greenend.chiark.sgtatham.puzzles.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm644 icons/mosaic-128d24.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/uk.org.greenend.chiark.sgtatham.puzzles.png
       - install -Dm644 icons/mosaic-128d24.png ${FLATPAK_DEST}/share/app-info/icons/flatpak/128x128/uk.org.greenend.chiark.sgtatham.puzzles.png
```

Comment:
```suggestion
```

This line should be removed, as the "suggestion" was.

Reply:
It fails to build without it

Reply:
I will repeat. This is not why it fails.